packages/sources/coinmetrics-test/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { PriceAdapter } from '@chainlink/external-adapter-framework/adapter'
+import { customSettings } from './config'
+import { price, burned, totalBurned } from './endpoint'
+
+export const adapter = new PriceAdapter({
+  defaultEndpoint: 'price',
+  name: 'COINMETRICS',
+  endpoints: [price, burned, totalBurned],
+  customSettings,
+  rateLimiting: {
+    tiers: {
+      community: {
+        rateLimit1m: 100,
+      },
+      paid: {
+        rateLimit1s: 300,
+      },
+    },
+  },
```

Comment:
Just want to confirm - no overrides?

Reply:
Yeah I think so. The original `coinmetrics` adapter has no adapter level overrides